### PR TITLE
Hal 797

### PR DESCRIPTION
**Pull request by zebrik**
Add tests for hierarchical composition of profiles.

### CODE REVIEW

**Comment by zebrik** [comment]
@rhatlapa  can you please review this?

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/profiles/ProfileOperations.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import org.jboss.hal.testsuite.dmr.AddressTemplate;
+import org.jboss.hal.testsuite.dmr.DefaultContext;
+import org.jboss.hal.testsuite.dmr.Dispatcher;
+import org.jboss.hal.testsuite.dmr.Operation;
+import org.jboss.hal.testsuite.dmr.StatementContext;
+
+/**
+ * Created by pjelinek on Aug 18, 2015
+ */
+public class ProfileOperations {
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/profiles/ProfileOperations.java:40]: Short javadocs for class as is this one would be great.

As this is class which is sharable for everyone needing to test parts concerning profiles.

File: common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java [hunk 2]
```diff
@@ -50,19 +50,36 @@
         }
     }
 
+    public interface Hook{
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java]: we should better enforce good code style, here is missing space before `{`

**Comment by zebrik** [comment]
I have force pushed requested changes.

**Comment by rhatlapa** [comment]
Thanks, merging

**Pull request merged**